### PR TITLE
Fixes from mmcint/synthoseis-testing + relative paths for directories

### PR DESCRIPTION
**Pull request by csamuel**
- Incorporate @mmcint fixes from https://github.com/mmcint/synthoseis-testing
- Update `environment.yml` to use python 3.9
- Ignore `.DS_Store` files
- Use relative paths to reference the config and project/work directories
- Add `output/` directory for generate model/work/temp files and ignore everything inside it

### CODE REVIEW

**Comment by csamuel** [comment]
The diff isn't readable, so here's the change to the notebook:

CleanShot 2023-07-25 at 19 17 22@2x

File: config/example.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "project": "example",
-  "project_folder": "/scratch/synthoseis_example",
-  "work_folder": "/scratch",
+  "project": "example1",
+  "project_folder": "./output/test4",
```
Comment (chadac) [inline config/example.json:3]: Is this a test artifact?

File: environment.yml [hunk 2]
```diff
@@ -7,12 +7,13 @@ dependencies:
   - h5py
   - matplotlib
   - numpy
-  - python=3.7.12=hf930737_100_cpython
+  - python=3.9
```
Comment (chadac) [inline environment.yml]: Betting this would work for 3.10 as well.

File: config/example.json [hunk 3]
```diff
@@ -16,10 +16,10 @@
   "bandwidth_ord": 4,
   "dip_factor_max": 2,
   "min_number_faults": 1,
-  "max_number_faults": 6,
+  "max_number_faults": 1,
   "pad_samples": 10,
   "max_column_height": [150.0, 150.0],
-  "closure_types": ["simple", "faulted", "onlap"],
```
Comment (mmcint) [inline config/example.json]: let's add 'faulted' and 'onlap' back in. i was just testing different cases

File: datagenerator/Closures.py [hunk 4]
```diff
@@ -863,7 +863,7 @@ def find_top_lith_horizons(self):
         self.top_lith_indices = np.array(top_lith_indices)
         self.top_lith_facies = self.facies[top_lith_indices]
 
-        # return top_lith_indices
```
Comment (mmcint) [inline datagenerator/Closures.py]: i was attempting to get the horizon tops by returning top lithology indices. we can ignore this change

File: main.py
Comment (mmcint) [inline main.py:1]: here i was just trying to debug and understand where the errors were occuring. we can leave in or take out. up to you @csamuel

**Review by mmcint — DISMISSED** [review]
there are a few changes, but for the most part it looks good

File: main.py
Comment (csamuel) [reply main.py:1]: Got it, will remove

**Comment by csamuel** [comment]
@mmcint Think I addressed all your feedback if you want to take another look

File: config/example.json (hunk 1 above)
Comment (mmcint) [reply config/example.json:3]: yes. but the `project` category is hard coded as `example1` in another script, so we should keep it in there

File: environment.yml (hunk 2 above)
Comment (mmcint) [reply environment.yml]: lets do it. i set this up in python 3.9 bc it was the earliest version i could get

**Review by mmcint — APPROVED** [review]
looks good. feel free to convert to python 3.10 @chadac

File: config/example.json (hunk 1 above)
Comment (csamuel) [reply config/example.json:3]: That actually reminds me, I meant ask what the story was behind checking for this specific `example1` project? https://github.com/studio-x-llc/synthoseis/blob/master/rockphysics/RockPropertyModels.py#L144-L149

**Review by chadac — APPROVED** [review]

File: config/example.json (hunk 1 above)
Comment (mmcint) [reply config/example.json:3]: i'm not sure i understand the if statement part, but essentially it's loading in the rock properties model (rock and fluid velocity properties) that is used to take the geomodel and convert to seismic

**Pull request merged**